scripts/hal_alignment/maf_to_fasta.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python3
+
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Convert each block of a multiple alignment format (MAF) file to FASTA format.
+
+Metadata are stored in a JSON file corresponding to each FASTA file.
```

Comment:
Open question: would it be worth saying the file will have the same name but different extension? (also applicable to the other places where you mention this file)

Reply:
Yes, I will note this.